products/amazon-rds-mysql.md

```diff
@@ -0,0 +1,64 @@
+---
+title: Amazon RDS for MySQL
+category: db
+iconSlug: mysql
+permalink: /rds-mysql
+releasePolicyLink: https://docs.aws.amazon.com/AmazonRDS/latest/UserGuide/MySQL.Concepts.VersionMgmt.html
+
+releases:
+-   releaseCycle: "8.0"
+    releaseDate: 2018-10-23
+    eol: 2026-04-01
+    latest: "8.0.32"
+    latestReleaseDate: 2023-02-07
+
+-   releaseCycle: "5.7"
+    releaseDate: 2016-02-22
+    eol: 2023-10-01
+    latest: "5.7.41"
+    latestReleaseDate: 2023-02-07
+
+-   releaseCycle: "5.6"
+    releaseDate: 2013-07-01
+    eol: 2022-03-01
+    latest: "unknown"
+
+---
+
+> [Amazon RDS for MySQL](https://aws.amazon.com/rds/mysql) is a PaaS offering from Amazon for
+> creating managed MySQL Community Edition databases. RDS makes it easier to set up, operate, and
+> scale MySQL deployments on AWS cloud.
+
+**MySQL recommends that all users run the latest available minor release for whatever major version
+is in use.**
+
+- AWS will provide support for major releases 3 years after their RDS release date.
+
+- AWS will provide support for minor versions 1 year after their RDS release date.
+
+Keep in mind that by default minor versions are automatically upgraded during maintenance windows.
+
+For more info on how RDS versions are deprecated see the AWS [documentation](https://aws.amazon.com/rds/faqs/#What_happens_when_an_Amazon_RDS_DB_engine_version_is_deprecated.3F).
+
+Please follow [best practices](https://aws.amazon.com/blogs/database/best-practices-for-upgrading-amazon-rds-for-mysql-and-amazon-rds-for-mariadb) when performing upgrades to your RDS instance.
```

Comment:
I do not think that's necessary : IMHO the main purpose of endoflife.date is to provide information on the support of a particular product, not on the upgrade best practices.

Reply:
I've removed this line now. As stated in another comment. I am just more simply stating in the section that there are risks.

Reply:
I'd be okay having this. We have similar links for other products where upgrades are considered tough/scary:

1. [AKS](https://endoflife.date/azure-kubernetes-service)
2. [NextCloud](https://endoflife.date/nextcloud#upgrades)
3. [GKE](https://endoflife.date/google-kubernetes-engine)